datafusion/sql/src/statement.rs

```diff
@@ -1163,20 +1165,40 @@ impl<'a, S: ContextProvider> SqlToRel<'a, S> {
 
     fn primary_key_from_constraints(
```

Comment:
🤔  maybe this function would be more clearly if it were renamed to reflect its expanded role for Constrants not just primary keys

Like

```suggestion
    fn constrants_from_constraints(
```

?